mke2fs.conf

```diff
@@ -2,46 +2,17 @@
 	base_features = sparse_super,large_file,filetype,resize_inode,dir_index
 	default_mntopts = acl,user_xattr
 	enable_periodic_fsck = 0
-	blocksize = 4096
-	inode_size = 256
-	inode_ratio = 16384
 
 [fs_types]
-	ext3 = {
-		features = has_journal
-	}
-	ext4 = {
+	# Based on ext4 with some features disabled
+	od-data = {
 		features = has_journal,extent
+		blocksize = 4096
 		inode_size = 256
-	}
-	small = {
-		inode_size = 128
-		inode_ratio = 4096
-	}
-	floppy = {
-		inode_size = 128
-		inode_ratio = 8192
-	}
-	big = {
-		inode_ratio = 32768
-	}
-	huge = {
-		inode_ratio = 65536
-	}
-	news = {
-		inode_ratio = 4096
-	}
-	largefile = {
-		inode_ratio = 1048576
-		blocksize = -1
-	}
-	largefile4 = {
-		inode_ratio = 4194304
-		blocksize = -1
-	}
-	hurd = {
-	     blocksize = 4096
-	     inode_size = 128
+		inode_ratio = 16384
```

Comment:
Maybe the inode ratio can even be a bit higher. OPK files are typically much larger than 16K since they have their own internal directory structure. Save games might be smaller or larger, depending on the game, but you're not going to fill an SD card with just save games. Even ROM images for 8-bit systems are often larger than 16K.

Reply:
Sure, changed to the `big` setting from `/etc/mke2fs.conf` which is 32768